pyHS100/pyHS100.py

```diff
@@ -103,7 +103,7 @@ def _query_helper(self, target, cmd, arg={}):
                 request={target: {cmd: arg}}
             )
         except Exception as ex:
-            raise SmartPlugException(ex)
+            raise SmartPlugException(ex) from ex
```

Comment:
I think we need to use ```raise_from``` in order to keep py27 compatibility.
See http://python-future.org/compatible_idioms.html#raising-exceptions